Documentation/ApiOverview/Localization/TranslationServer/Crowdin/ExtensionIntegration.rst

```diff
@@ -45,86 +45,167 @@ Crowdin:
 Step-by-step instructions for GitHub
 ------------------------------------
 
+..  rst-class:: bignums-xxl
 
-.. _crowdin-extension-integration-github-crowdin-config:
```

Comment:
These anchors are permalinks that might have been linked from somewhere. They must not be removed. You can move them as addtional anchors with the headline in line 45 or just keep the headers. 

In general headers create a semantic section that can be searched for as separate block. Therefore I would suggest to keep the existing structue and only do content changes where needed.



Your PR is doing way more then what the title says?